services/autoscaling/tests/unit/conftest.py

```diff
@@ -40,12 +75,385 @@ def app_environment(
     envs = setenvs_from_dict(
         monkeypatch,
         {
-            "AWS_KEY_NAME": "TODO",
-            "AWS_DNS": faker.domain_name(),
-            "AWS_ACCESS_KEY_ID": "str",
-            "AWS_SECRET_ACCESS_KEY": "str",
-            "AWS_SECURITY_GROUP_IDS": '["a", "b"]',
-            "AWS_SUBNET_ID": "str",
+            "EC2_ACCESS_KEY_ID": faker.pystr(),
+            # "EC2_ENDPOINT": "null",
+            "EC2_SECRET_ACCESS_KEY": faker.pystr(),
+            "EC2_INSTANCES_KEY_NAME": faker.pystr(),
+            "EC2_INSTANCES_SECURITY_GROUP_IDS": json.dumps(
+                faker.pylist(allowed_types=(str,))
+            ),
+            "EC2_INSTANCES_SUBNET_ID": faker.pystr(),
+            "EC2_INSTANCES_AMI_ID": faker.pystr(),
+            "EC2_INSTANCES_ALLOWED_TYPES": json.dumps(
+                faker.pylist(allowed_types=(str,))
+            ),
         },
     )
     return mock_env_devel_environment | envs
+
+
+@pytest.fixture(scope="function")
```

Comment:
why is this specified, thought the default was function.

Reply:
it's copy pasted from somewhere else. you are right this is useless.